pcsx2/USB/qemu-usb/vl.h

```diff
@@ -25,64 +25,38 @@
 #define VL_H
 
 /* we put basic includes here to avoid repeating them in device drivers */
-#include <stdlib.h>
-#include <stdio.h>
-#include <stdarg.h>
-#include <string.h>
-
-typedef signed __int8 int8_t;
-typedef signed __int16 int16_t;
-typedef signed __int32 int32_t;
-typedef signed __int64 int64_t;
-typedef unsigned __int8 uint8_t;
-typedef unsigned __int16 uint16_t;
-typedef unsigned __int32 uint32_t;
-typedef unsigned __int64 uint64_t;
-
+#include <cstdlib>
+#include <cstdio>
+#include <cstdarg>
+#include <cstring>
+#include <cstdint>
+#include <cstddef>
 #include <limits.h>
 #include <time.h>
 #include <ctype.h>
-#include <errno.h>
+#include <cassert>
+#include <cerrno>
 #include <fcntl.h>
 #include <sys/stat.h>
 
+#if !defined(_MSC_VER)
 #define inline __inline
-
-#ifndef O_LARGEFILE
-#define O_LARGEFILE 0
-#endif
-#ifndef O_BINARY
-#define O_BINARY 0
 #endif
 
 #ifdef _WIN32
-#include <windows.h>
-#define fsync _commit
-//#define lseek _lseeki64
-//#define ENOTSUP 4096
-extern int qemu_ftruncate64(int, int64_t);
-#define ftruncate qemu_ftruncate64
-
-
-static inline char *realpath(const char *path, char *resolved_path)
-{
-    _fullpath(resolved_path, path, _MAX_PATH);
-    return resolved_path;
-}
-
 #define PRId64 "I64d"
```

Comment:
I'm getting macro redefinitions for these, are they still used because I can't seem to find references.